src/functions/severa/get-flextime-by-user/handler.ts

```diff
@@ -0,0 +1,38 @@
+import type { APIGatewayProxyEvent, APIGatewayProxyHandler } from "aws-lambda";
+import type FlextimeModel from "src/database/models/severa";
+import { CreateSeveraApiService } from "src/database/services/severa-api-service";
+
+/**
+ * @param event
+ * @returns users flextime
+ */
+export const getFlextimeHandler: APIGatewayProxyHandler = async (
```

Comment:
It seems for some functions we set the return type as APIGatewayProxyHandler and for others we set it as ValidatedEventAPIGatewayProxyEvent<any>. Please investigate the two and then change this accordingly.

It looks to me like ValidatedEventAPIGatewayProxyEvent<any> is the way it's done in Elastic-app-functions which should be our example.
Also in most cases the any can be replaced with the typeof schema- see schame/questionnaire.ts for an example. Might be worth discussing this with Ilmo as to whether the questionnaire lambdas should also be updated.